src/m365/spo/commands/site/site-commsite-enable.ts

```diff
@@ -46,6 +53,9 @@ class SpoSiteCommSiteEnableCommand extends SpoCommand {
       },
       {
         option: '-i, --designPackageId [designPackageId]'
+      },
+      {
+        option: '--designPackage [designPackage]'
```

Comment:
Let's use Autocomplete functionality here.

Also: shall we add a -p short option?